projects/core/src/product/store/effects/product-search.effect.ts

```diff
@@ -32,7 +30,7 @@ export class ProductsSearchEffects {
               catchError(error =>
                 of(
                   new productsSearchActions.SearchProductsFail(
-                    error,
+                    makeHttpErrorSerializable(error),
```

Comment:
Just want to ask since references / reviews aren't in this pr and I want to make a connection from this and the leftovers 

--> this action takes a payload: ErrorModel and referencesFail takes that as well, so can't we pass makeHttpErrorSerializable(error) to the product references effect as well, even though it has this right now 

new productReferencesActions.LoadProductReferencesFail({
                message: payload.productCode,
              } as ErrorModel)

but I think it can be easily changed to .LoadProductReferencesFail(makeHttpErrorSerializable(error));

Reply:
No need to do that, since `ErrorModel` is already serializable. We could of course, and the `makeHttpErrorSerializable` would return the provided object unmodified, but no need for that.